mobify/progressive-web-scaffold
# Astro: Merlins fix account tab + onboarding from the merge + cart item added modal

Fixes broken components from the dev merge into `astro-merlins-base`

## Changes
- login container import `isRunningInAstro` directly
- onboarding buttons trigger astro
- coverage is calculated and passed through

## How to test-drive this PR
- test the account page: switch between register tab and sign in tab
- from onboarding selecting register now and sign in should lead you to the account tab
